pallets/oracle-data-collection/src/lib.rs

```diff
@@ -460,17 +541,58 @@ pub mod types {
 		}
 	}
 
+	/// Information of a collection
+	#[derive(
+		Encode, Decode, PartialEq, Eq, Clone, TypeInfo, RuntimeDebugNoBound, MaxEncodedLen,
+	)]
+	#[scale_info(skip_type_params(T))]
+	pub struct CollectionInfo<T: Config> {
+		/// Maximum duration to consider an oracle value non-outdated.
+		/// An oracle value is consider updated if its timestamp is higher
+		/// than `now() - value_lifetime`
+		pub value_lifetime: Option<T::Timestamp>,
+
+		/// Minimun number of feeders to succesfully aggregate a value.
+		pub min_feeders: u32,
+	}
```

Comment:
To catch all casuistics well, I've done a new type that is used as a parameter for the new extrinsic `set_collection_info()` to configure a collection with some properties.

`min_feeders` force to have at least `min_feeders` per key in a collection. That means that the feeder should have fed and should have fed recently if `value_lifetime` is different than `None`